circleci/provider.go

```diff
@@ -26,6 +26,12 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("CIRCLECI_ORGANIZATION", nil),
 				Description: "The CircleCI organization.",
 			},
+			"url": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("CIRCLECI_URL", "https://circleci.com/api/v1.1"),
```

Comment:
As v1.1 is the current stable API version I would say it's fine for now to hardcode it.

If v2 API is promoted to stable at some point or if we need API >=v2 for some resources we may need to change this default value.

Reply:
I did wonder where this should be set. I was in two minds as to whether this should be in the provider of the client. However, seeing as the client is currently not maintained I thought it best to be here